### ISSUE TITLE
[Suggestion] Backend Developer

### ISSUE THREAD

**Issue by DeepKotadiya144** [kamranahmedse/developer-roadmap#1684]
Can you please choose your personal favourite one language in backend development?
Because there is no personal preference in language tag.No purple tick😀
Thank you for everything 🙏

**Comment by Olayanju-1234** [comment]
Yes. You can pick a language that ypu are most comfortable with. Python has django for backend and Javascript has Nodejs. There is also PHP.

**Comment by DhirajsGithub** [comment]
Javascript

**Comment by Jaya-sys** [comment]
What do i have to make to contribute in this project ?
